src/adapters/poi/idunn_poi.js

```diff
@@ -53,9 +53,9 @@ export default class IdunnPoi extends Poi {
     }
   }
   /* ?bbox={bbox}&category=<category-name>&size={size}&verbosity=long/ */
-  static async poiCategoryLoad(bbox, size, category, query) {
+  static async poiCategoryLoad(bbox, size, category, query, extendBBox = false) {
     const url = `${serviceConfig.idunn.url}/v1/places`;
-    const requestParams = { bbox, size };
+    const requestParams = { bbox, size, extend_bbox: extendBBox };
```

Comment:
Really minor thing: should we prefer `extendBBox` or `extendBbox` with a lowercase 'b' ? Like `parseBboxString`

Reply:
Looks like we have more `Bbox` than `BBox` already, which is consistent with the frequent use of the standalone `bbox`. So I'll standardize these cases to `Bbox`.
Thanks for making this point :)